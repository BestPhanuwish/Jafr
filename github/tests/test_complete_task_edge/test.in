1
0
3
abdjskabj
0 1 2
1 3
1 2 3
1 2a
a 1
1
1
2
1 1 1 1
6