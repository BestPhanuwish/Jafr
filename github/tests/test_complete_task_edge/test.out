Just a friendly reminder! You have these tasks to finish today.

These tasks need to be finished in the next three days!

You have the following meetings today!

You have the following meetings scheduled over the next week!

What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
Which task(s) would you like to mark as completed?
1. next week task by 08/08/23
2. reverse ideology by 31/07/23
Please pick a sequence of number between 1-2
Please pick a sequence of number between 1-2
Please pick a sequence of number between 1-2
Please pick a sequence of number between 1-2
Please pick a sequence of number between 1-2
Please pick a sequence of number between 1-2
Please pick a sequence of number between 1-2
Please pick a sequence of number between 1-2
Marked as complete.
What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
Which task(s) would you like to mark as completed?
1. reverse ideology by 31/07/23
Please pick a sequence of number between 1-1
Marked as complete.
What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
